### PR TITLE
Update free-programming-books-zh.md

### PR DESCRIPTION
**Pull request by HaimmingYu**
fixed  [nutzbook](https://github.com/EbookFoundation/free-programming-books/issues/2529)

## What does this PR do?
Add Resource

### Description 
Nutz -- Web Framework(Mvc/Ioc/Aop/Dao/Json) for ALL Java developer https://nutzam.com

### Why is this valuable (or not)
 Web Framework(Mvc/Ioc/Aop/Dao/Json) for ALL Java developer

### How do we know it's really free?
This work is licensed by using knowledge sharing signature - sharing the 4 international license agreement in the same way. Visit
 http://creativecommons.org/licenses/by-sa/4.0/

### For book lists, is it a book?
yes

### Checklist:
- [ ] Version v1.8-20161219
- [ ] The book's source code is hosted in [GitHub](https://github.com/wendal/nutz-book) and [open source China](http://git.oschina.net/wendal/nutz-book).
- [ ] The hosting address of the book is also in [GitHub](https://github.com/wendal/nutz-book) and [open source China](http://git.oschina.net/wendal/nutz-book).
- [ ] Wendal Chen write Fisrt Commit on Mar 9, 2015

### CODE REVIEW

**Comment by eshellman** [comment]
Almost there, just switch the order of the two links.

**Comment by eshellman** [comment]
thanks for addressing #2529

**Comment by HaimmingYu** [comment]
What shall I do?

**Comment by eshellman** [comment]
switch the order of the two links

File: free-programming-books-zh.md [hunk 1]
```diff
@@ -475,12 +475,12 @@
 * [MyBatis中文文档](http://mybatis.github.io/mybatis-3/zh/index.html)
 * [Netty 4.x 用户指南](https://github.com/waylau/netty-4-user-guide)
 * [Netty 实战(精髓)](https://github.com/waylau/essential-netty-in-action)
+* [Nutz文档](https://nutzam.com/core/nutz_preface.html)
+* [Nutz-book Nutz烹调向导](http://nutzbook.wendal.net)
```
Comment (bluetata) [inline free-programming-books-zh.md]: you can try to move ` [Nutz-book Nutz烹调向导](http://nutzbook.wendal.net)` up to ` [Nutz文档]`

like this:
```markdown
* [Nutz-book Nutz烹调向导](http://nutzbook.wendal.net)
* [Nutz文档](https://nutzam.com/core/nutz_preface.html)
```

Comment (bluetata) [reply free-programming-books-zh.md]: 还是机器check有问题，你添加的那两个顺序互换一下应该就可以了

**Comment by eshellman** [comment]
Thanks @HaimmingYu and welcome to free-programming-books. 谢谢 @bluetata .

**Pull request merged**